cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/metrics/MetricsCollector.java

```diff
@@ -166,8 +160,13 @@ private Measurement createMeasurement(String service, double value) {
             final List<MBeanAttributeInfo> filteredAttributes = new ArrayList<>();
 
             for (MBeanAttributeInfo attributeInfo : attributes) {
-                if (allowedTypes.contains(attributeInfo.getType())) {
+                try {
+                    final Object obj = mbeanServerConn
+                            .getAttribute(objectInstance.getObjectName(), attributeInfo.getName());
+                    Double.parseDouble(obj.toString());
```

Comment:
I'll add comments. The reason behind the ugly try/catch block is that some attributes are marked as type Object but they have a number value (thats why I missed some of the metrics by going with numeric value types only). Someone made mbeans in cassandra expose object type values for counters, percentages, etc